### ISSUE TITLE
Verify inpainting results

### ISSUE THREAD

**Issue by wangherr** [CompVis/latent-diffusion#243]
Using the pretrained checkpoint, I choose an image and a mask to do inpainting. However, the result is bad. 
Is the bad result caused by the difference between the inquired image and the training dataset？
``
python scripts/inpaint.py --indir data/inpainting_examples/ --outdir outputs/inpainting_results
``

input:
![n02381460_2870](https://user-images.githubusercontent.com/68674487/222941356-9aa30016-46b6-40a5-818d-bb61e453496d.png)

mask:
![n02381460_2870_mask](https://user-images.githubusercontent.com/68674487/222941359-836829e7-9512-4d7d-8aaf-c9cf9440d236.png)

result under "--steps 50":
![n02381460_2870_step50](https://user-images.githubusercontent.com/68674487/222941360-7f10a024-4bb5-47b2-b587-3d3c6f8339e8.png)

result under "--steps 500":
![n02381460_2870_step500](https://user-images.githubusercontent.com/68674487/222941361-c20c0919-cbf4-45aa-a21f-fd7766a71fe2.png)

**Comment by aleksmirosh** [comment]
@wangherr 
did you try to run with just a standard image?

**Comment by wangherr** [comment]
> @wangherr did you try to run with just a standard image?

Yes, the standard image is ok.
